tsconfig.build.json

```
@@ -0,0 +1,36 @@
+/* Visit https://aka.ms/tsconfig.json to read more about this file */
```

shortened URL makes me uneasy; was this generated? if not, how about...

```suggestion
/* Visit https://www.typescriptlang.org/tsconfig to read more about this file */
```

---

I think it's the default config that TS makes. I'm fine changing it